apps/commercetools-without-search/src/config.ts

```diff
@@ -79,5 +79,9 @@ export function validateParameters(parameters: ConfigurationParameters): string
     return 'Provide the commercetools data locale.';
   }
 
+  if (parameters.authApiEndpoint?.endsWith('/')) {
+    parameters.apiEndpoint = parameters.apiEndpoint?.slice(0, -1);
```

Comment:
Is this correct to check `authApiEndpoint` and then overwrite `apiEndpoint`?

Reply:
No, this is copy paste error. Pushed the right commit now, sorry.